.github/workflows/publish-module-api-doc.yaml

```diff
@@ -4,7 +4,11 @@ on:
   push:
     branches:
       - 'master'
+    paths:
```

Comment:
Can we have these changes in a separate commit?

Reply:
Sure, splitted.